src/auxinfo/participant.rs

```diff
@@ -522,6 +598,8 @@ impl AuxInfoParticipant {
         debug!("Creating new auxinfo.");
 
         let (decryption_key, p, q) = DecryptionKey::new(rng)?;
+        // XXX should this be `self.retrieve_context()` instead of `&()`? I
+        // realize they are the same now, but may change in the future?
```

Comment:
Yes, it should be! Rawane fixed this in [an open PR](https://github.com/boltlabs-inc/tss-ecdsa/pull/280/files#diff-8464b562e103bfbf3b9654e47a7c24ee39f48127ee34eca29318e310527f505cR527) but you can also fix it here to avoid a merge conflict mistake if you want.